### PR TITLE
mgr/cephadm: Deploying a cluster with the Vagrantfile fails

### PR DESCRIPTION
**Pull request by votdev**
- Get the deployment of a small Ceph cluster for cephadm development running again by switching to the 'centos/stream8' box. 
- Adapt URL to Ceph repo.
- Do not install the non-existing/removed `ceph` package.
- Install cephadm
- Install `libseccomp-devel` (to prevent the `podman: symbol lookup error: podman: undefined symbol: seccomp_notify_fd` error)

Fixes: https://tracker.ceph.com/issues/55673

Signed-off-by: Volker Theile <vtheile@suse.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [x] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by votdev** [comment]
jenkins retest this please

**Comment by votdev** [comment]
Please note, the failed tests are not related to this PR.

**Comment by votdev** [comment]
> dumb question, I might miss a detail but why don't we use CentOS Stream 8 instead?
> 
> shouldn't we change the following ? (line 29) from
> 
> ```
> config.vm.box = "centos/8"
> ```
> 
> to
> 
> ```
> config.vm.box = "centos/stream8"
> ```

I don't know nor using CentOS But i can give it a try.

**Comment by guits** [comment]
jenkins test api

File: src/pybind/mgr/cephadm/Vagrantfile [hunk 1]
```diff
@@ -61,6 +61,6 @@ Vagrant.configure("2") do |config|
     sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
     sudo rpm --import 'https://download.ceph.com/keys/release.asc'
     curl -L https://shaman.ceph.com/api/repos/ceph/master/latest/centos/8/repo/ | sudo tee /etc/yum.repos.d/shaman.repo
-    sudo yum install -y python36 podman ceph
+    sudo yum install -y python36 podman lvm2 libseccomp-devel
```
Comment (guits) [inline src/pybind/mgr/cephadm/Vagrantfile:64]: shouldn't we add `cephadm` package in this list?

**Comment by votdev** [comment]
jenkins test windows

**Review by guits — APPROVED** [review]

**Pull request merged**